sdk/react/bottomOverlay.js

```diff
@@ -100,6 +100,12 @@ var BottomOverlay = React.createClass({
     }
   },
 
+  componentWillReceiveProps: function(nextProps) {
+    if (Math.abs(this.props.playhead - nextProps.playhead) >= 0.2) {
```

Comment:
We can't just do "!=" because then the cachedPlayhead will always be set to -1 and thus never be used.  I propose to define a var scrubPositionDelta = 0.2 and to add a comment before the function.